imxrt-hal/src/iomuxc/macros.rs

```diff
@@ -2,13 +2,9 @@ macro_rules! alt0 {
     ($Pad:ident, $mux_mod:ident) => {
         /// Converts the pad into its Alt0 setting
         pub fn alt0(self) -> $Pad<$crate::iomuxc::Alt0> {
-            self.iomuxc().$mux_mod.modify(|_, w| {
-                w.mux_mode()
-                    .variant($crate::ral::iomuxc::$mux_mod::MUX_MODE_A::ALT0)
-            });
+            unsafe { modify_reg!($crate::ral::iomuxc, $crate::ral::iomuxc::IOMUXC, $mux_mod, MUX_MODE: $crate::ral::iomuxc::$mux_mod::MUX_MODE::RW::ALT0) };
```

Comment:
The `use imxrt_ral::modify_reg;` imports throughout the `gpio` files could be removed if the `modify_reg!` macro were absolutely referenced. How about

```
::imxrt_ral::modify_reg!($crate::ral::iomuxc, ...)
```

instead of `modify_reg!`? Comment applies for each `modify_reg!` usage in this file.

Reply:
Perfect, yeah I'm still so used to pre-2018 where everything had to be extern use to get macros in, I never fully learned the new rules beyond use xyz::somemacro;

Neat, will fix